make/ibm.mk

```diff
@@ -1,31 +1,54 @@
-VERSION ?= 1.0.0
-
-BUNDLE_IMG ?= ibm-account-iam-operator-bundle:v$(VERSION)
-
-IMG ?= ibm-account-iam-operator:v$(VERSION)
+DEV_VERSION ?= dev # Could be other string or version number
+DEV_REGISTRY ?= quay.io/bedrockinstallerfid
 
 YQ_VERSION ?= v4.44.1
 
-# change the image to dev when applying deployment manifests
-deploy: IMG = quay.io/bedrockinstallerfid/ibm-account-iam-operator:dev
+ifneq ($(shell echo "$(DEV_VERSION)" | grep -E '^[^0-9]'),)
+	TAG := $(DEV_VERSION)
+else
+	TAG := v$(DEV_VERSION)
+endif
```

Comment:
I don't see a difference between the use of `TAG` and `VERSION` variables. Why not just keep using `VERSION`?

Reply:
I think the reason is If the version is a string like: "dev", or "latest", the final tag will be `vdev`, `vlatest`, as it always has letter v in front on a version number.

Reply:
Alternatively, we might set `VERSION ?= v0.0.1` with `v` directly, eliminating the need for the `TAG` variable.